docs/surveys/2019-10_3-months-of-govuk-docker.md

```diff
@@ -0,0 +1,291 @@
+# 3 months of GOV.UK Docker Survey
```

Comment:
I was slightly confused by the file name. Does it need to contain the year/month?

Reply:
I want to set a precedent where surveys are chronologically ordered by default in directory listing. I've amended the name to try make it be a little less special.